packages/landing/package.json

```diff
@@ -36,7 +36,7 @@
     "@servie/events": "^3.0.0",
     "@types/geojson": "^7946.0.7",
     "@types/proj4": "^2.5.0",
-    "maplibre-gl": "^1.14.1-rc.2",
+    "maplibre-gl": "^2.1.9",
```

Comment:
That's a big jump of version. Hopefully not broken anything.

Reply:
Might be support global view now? https://www.mapbox.com/blog/globe-view/?utm_campaign=email%7Cblast%7Cglobe-launch-weather&utm_content=Globe+view+announcement+1&utm_medium=email_action&utm_source=customer.io

Reply:
I think it went 1.15 -> 2.0.0 then 2.1.0 then a bunch of bug fixes, mostly looks to be around types the brokenness.

it did fix some of the loading event problems